public/tabs/chat/chat_page.tsx

```diff
@@ -21,7 +21,9 @@ export const ChatPage: React.FC<ChatPageProps> = (props) => {
   const chatContext = useChatContext();
   const { chatState, chatStateDispatch } = useChatState();
   const conversationLoadStatus = useObservable(core.services.conversationLoad.status$);
-  const conversationsStatus = useObservable(core.services.conversations.status$);
+  const conversationsStatus = useObservable(
+    core.services.conversationLoad.conversationsService.status$
```

Comment:
Minor: it's not related to this change directly, but could we have a better naming for `ConversationService` and `ConversationLoadService`? I somehow think the currently naming is a bit confusing, for example, in this line the `services.conversationLoad.conversationsService` sounds a bit weird.

Reply:
What about exposing only a status observable instead of the whole conversations service? The `conversationsService` can be a private property of the conversation load service. The `latestIdStatus` will be exposed to determine the current status.